backend/pom.xml

```diff
@@ -14,6 +14,7 @@
     <properties>
         <assertj.version>3.15.0</assertj.version>
         <aws.ses.version>1.11.789</aws.ses.version>
+        <aws.s3.version>1.11.848</aws.s3.version>
```

Comment:
try to use the same AWS SDK for all AWS services you integrate to avoid incompatibilities . SES, S3, ...